tdrs-backend/docs/api/authentication.md

```diff
@@ -4,3 +4,14 @@ For clients to authenticate, they have to authenticate with Login.gov via the ba
  This will allow the backend to identify the browser which requested access and authorize them based on the cookie they provide in their API calls. 
 
  The secured portion of this authorization is due to the httpOnly cookie being inaccessible to the client's local browser. 
+
+# Generating API token
+
+In order to use APIs, the `OFA Admin` user has to generate a new token and use it in the API request following these steps:
+1. User has to first login using frontend and going through the normal login process
+2. After user is logged in, user can grab a token at `/v1/security/get-toke`
```

Comment:
```suggestion
2. After user is logged in, user can grab a token at `/v1/security/get-token`
```

I assume it was just a typo. Does it need a trailing `/` as well?

Reply:
Yes, and doesn't need the trailing `/`